### ISSUE TITLE
Integrate PersistedAliasMapperLocales

### ISSUE THREAD

**Issue by klodeckl** [calien666/typo3-extended-routing#2]
Hi,
would you integrate a PersistedAliasMapperLocales? 
Here is the link to the file: https://gist.github.com/klodeckl/08ab910b3ec6ef3e4acb2bed1c986406

I’ve attached a working file. But it needs some changes, line 247-248: locales and db field is still hardcoded there.

**Comment by calien666** [comment]
I'll try to get your intention behind your feature request.
You want to get an AliasMapper for non translated tables with multiple slugs included? For example, language is set to -1, the dataset is always the same, but for every needed language, you'll add a custom slug field?

I'm trying to figure out, which issues will lead to this. Could you give me more information about your intentions?

**Comment by klodeckl** [comment]
Yes: I have an extension with one record for all languages (german and english fields are in one record). And therefore two slug fields are needed. The reason is because the records are inserted by frontend users (extbase new/create/edit/update action) and for simplicity to the editors everything is inserted/edited in one form in the frontend.

**Comment by SventB** [comment]
This would be useful for EXT:static_info_tables and table "static_countries". I'd like to introduce two (or more) columns for slugs there and use it for different languages.
